src/components/box.js

```diff
@@ -97,6 +106,21 @@
       </Box>
     );
 
+    const Component =
+      hasLink || hasExternalLink ? (
+        <Link
+          href={hasExternalLink ? linkToExternalText : undefined}
```

Comment:
at this point we already checked if `hasExternalLink` exists, we can just use `linkToExternalText`